chrome/content/zotero/zoteroPane.js

```diff
@@ -175,14 +167,60 @@ var ZoteroPane = new function()
 		const buttons = toolbar.getElementsByTagName("toolbarbutton");
 		const focusMap = new Map();
 		const zones = [
-			new TbFocusZone(
-				"zotero-tb-collection-add", 
-				null, 
-				"zotero-tb-search-menu-button"),
-			new TbFocusZone(
-				"zotero-tb-locate", 
-				"zotero-tb-search",
-				null)
+			{
+				get start() { return document.getElementById("zotero-tb-collection-add"); },
+				focusBefore() {
+					// need to check if the itembox is open
+					const pane = document.getElementById("zotero-item-pane-content");
+					if (pane.selectedIndex === "0") {
```

Comment:
Remove comment above and add "// If no item is selected, focus items list" above this line